packages/api/endpoints/rules.js

```diff
@@ -153,24 +269,9 @@ async function put(req, res) {
     const oldRule = await rulePgModel.get(knex, { name });
     const oldApiRule = await translatePostgresRuleToApiRule(oldRule, knex);
 
-    apiRule.updatedAt = Date.now();
-    apiRule.createdAt = oldRule.created_at;
```

Comment:
Fixed this: oldRule.created_at is Date type while createdAt is number. Schema validation reveals this issue.